dallinger/experiment_server/dashboard.py

```diff
@@ -696,11 +703,61 @@ def database():
         for c in datatables_options.get("columns", [])
         if c.get("data")
     ]
+
+    # Extend with custom actions
+    actions = {
+        "extend": "collection",
+        "text": "Actions",
+        "buttons": [],
+    }
+    buttons = actions["buttons"]
+
+    exp_actions = exp.dashboard_database_actions()
+    for action in exp_actions:
+        buttons.append(
+            {
+                "extend": "route_action",
+                "text": action["title"],
+                "route_name": action["name"],
+            }
+        )
+
+    is_sandbox = getattr(recruiters.from_config(get_config()), "is_sandbox", None)
```

Comment:
Yeah, I'd still need this code because I need to know if we're in a sandbox for the command, but it's probably not the best way to determine whether the action is present. It does the job for now though.